public_html/documentation.php

```diff
@@ -45,27 +45,78 @@ function build_sidebar_nav($elements) {
     global $md_fn;
     foreach ($elements as $name => $element) {
         if (!isset($element['url'])) {
-            $path = explode('/', $element[0]['url']);
-            $show = strpos($md_fn, implode('/', array_slice($path, 0, -1))) !== false ? 'show' : '';
-            $is_open = $show == 'show' ? 'true' : 'false';
-            $id = str_replace(' ', '-', strtolower(implode('-', array_slice($path, -3, 2))));
-            $id = str_replace(':', '', $id);
-            $sidebar_nav .=
-                '<button class="btn d-inline-flex align-items-center" data-bs-toggle="collapse" data-bs-target="#' .
-                $id .
-                '" aria-expanded="' .
-                $is_open .
-                '" aria-current="' .
-                $is_open .
-                '">
-                            <i class="fas fa-angle-right me-3"></i><strong>' .
-                ucwords($name) .
-                '
-                        </strong></button>';
-            $sidebar_nav .=
-                '<nav class="collapse ' . $show . '" id="' . $id . '"><ul class="list-unstyled fw-normal ps-3 small">';
-            build_sidebar_nav($element);
-            $sidebar_nav .= '</ul></nav>';
+            # Build landing pages
```

Comment:
I don't think this split is needed anymore.

Reply:
Hm not sure I understand, you mean that copying the code for the if statements `if($_GET['path'] === 'usage'){` and `elseif($_GET['path'] === 'contributing'){` from the other pages isn't needed?

I thought so too, but if I remove any line the code breaks 😬 I was trying really hard not to have to duplicate the code so much, but couldn't get it to work, not sure why 🤔

Reply:
hmm, let me have a look

Reply:
the problem was that the path was `/usage|contribuing` and not `/docs/usage|contribuing`. I fixed this now in the now in the most recent commit